facebook_events_importer.js

```diff
@@ -0,0 +1,74 @@
+// ==UserScript==
+// @name        MusicBrainz: Import events from Facebook
+// @version     2016-01-17
+// @author      Ohm Patel
+// @require     https://code.jquery.com/jquery-2.2.0.min.js
+// @grant       none
+// @include     *.facebook.*/events/*
```

Comment:
I would match protocol too: `http*://*.facebook.*/events/*`
Also this expression may match `https://www.facebook.com/anything/something/anotherthing/events/things/stuff`, you may want to use stricter regex as shown at https://github.com/murdos/musicbrainz-userscripts/blob/master/bandcamp_importer.user.js#L8 for example.